File: typescript/cognito-api-lambda/src/helloworld.ts
export const handler = async (_event: any = {}): Promise<any> => {
  return {
    statusCode: 200,
    headers: { 'Content-Type': 'application/json' },
    body: JSON.stringify({
      service: 'cognito-api-lambda',
      runtime: 'nodejs22.x',
      region: process.env.AWS_REGION,
      functionName: process.env.AWS_LAMBDA_FUNCTION_NAME,
      timestamp: new Date().toISOString(),
    }),
  };
};

File: java/cognito-api-lambda/src/main/resources/lambda/hello-world.py
import json

def handler(event, context):
    return {
        'statusCode': 200,
        'headers': {'Content-Type': 'application/json'},
        'body': json.dumps({
            'service': 'cognito-api-lambda',
            'runtime': 'python3.12',
            'framework': 'aws-cdk',
            'language': 'java'
        })
    }
